.github/workflows/pre_commit_and_sanity.yml

```diff
@@ -23,10 +23,16 @@ on:
         description: The collection url
         required: false
         type: string
+      collection_dependencies:
+        description: The collection dependencies (as a space separated list)
+        required: false
+        type: string
 
 jobs:
   pre-commit:
     uses: "redhat-cop/ansible_collections_tooling/.github/workflows/pre-commit.yml@main"
+    with:
+      collection_dependencies: ${{ inputs.collection_dependencies }}
```

Comment:
if you are going to add required files to the pre-commit, you need to add them here with collection dependencies like like 39-42 for sanity.